components/gc-chckbxrdio/_base.scss

```diff
@@ -15,6 +15,11 @@
 	box-shadow: 0 0 0 2px #000, 0 0 8px 4px rgb(59, 153, 252);
 }
 
+fieldset.gc-chckbxrdio {
+	border-top: 0;
+	padding-top: 0;
+}
+
```

Comment:
@duboisp You decided to include the `chkbxrio-grp` styles so that we don't need to use the helper? Good idea.